src/commands/remove_funds.rs

```diff
@@ -8,15 +8,32 @@ use num_traits::Zero;
 use signatory::FsKeyStore;
 
 use crate::{
-    cellar_uniswap_wrapper::{UniswapV3CellarRemoveParams, UniswapV3CellarState, UniswapV3CellarTickInfo},
+    cellar_uniswap_wrapper::{UniswapV3CellarState, UniswapV3CellarTickInfo},
     prelude::*,
     uniswap_pool::PoolState,
 };
+use rebalancer_abi::cellar_uniswap::*;
 
 /// Remove funds from Cellars
 #[derive(Command, Debug, Clap)]
 pub struct RemoveFundsCmd {}
 
+pub fn cellar_remove_params(
```

Comment:
Same here, the caller could directly instantiate the struct.

Reply:
Fixed with this PR: https://github.com/PeggyJV/cellar_rebalancer_rs/pull/33/commits/8f4daaad16ad94376c5ef59c50719b0a22b34c9f